sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -14,13 +15,21 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
     },
 
     markAllAsRead: function() {
+        this.stopListening(this.collection, 'change:unread');
+        this.model.save({unread: false});
```

Comment:
Yes, but there's no point for it to get hit 100 times for every item in a feed. Instead, I have it stopListening and just call it manually here (which is better I think).